src/plugins/CaptionsPlugin.js

```diff
@@ -117,25 +278,74 @@ export class CaptionsPlugin extends ButtonPlugin {
   }
 
   /**
-   *
-   *
    * @memberof CaptionsPlugin
    */
   captionsButtonClick() {
     this.captionsMuted = !this.captionsMuted;
 
     for (let i = 0; i < this.captionsButtonLength; i ++) {
-      this._captionsButtons[i].button.dataset['captionsMuted'] = this.captionsMuted;
+      this._captionsButtons[i].button.classList.add(this.captionsMuted ? 'muted' : 'unmuted');
     }
   }
 
+  /**
+   * Fired whenever the font size radios are updated
+   * @param {Event} e
+   * @memberof CaptionsPlugin
+   */
+  onFontSizeChange(e) {
+    this.setCaptionsStyles('size', e.target.value);
+
+    this.fontSizeRadios.forEach((group) => {
+      group.radioGroup[e.target.value].checked = true;
+    });
+  }
+
+  /**
+   * Fired whenever the alignment radios are updated
+   * @param {Event} e
+   * @memberof CaptionsPlugin
+   */
+  onAlignmentChange(e) {
+    this.setCaptionsStyles('align', e.target.value);
+
+    this.alignmentRadios.forEach((group) => {
+      group.radioGroup[e.target.value].checked = true;
+    });
+  }
+
+  /**
+   * Fired whenever the color radios are updated
+   * @param {Event} e
+   * @memberof CaptionsPlugin
+   */
+  onColorChange(e) {
+    const styles = e.target.value === 'default' ? {color: 'white', background: 'black'} : {color: 'black', background: 'white'};
```

Comment:
Let's use some variables to clean it up tad:

```javascript
const DEFAULT_CAPTION_STYLE = { color: 'white', background: 'black' };
const INVERTED_CAPTION_STYLE = { color: 'black', background: 'white' };

const styles = e.target.value === 'default' ? DEFAULT_CAPTION_STYLE : INVERTED_CAPTION_STYLE;
```

Reads a little more explicit.